### PR TITLE
Tidbits Site home page improvements

### PR DESCRIPTION
**Pull request by MSamiTariq**
# Pull Request Template

Before submitting this PR, please make sure you have completed the following checklist. This helps ensure that our codebase remains high quality and that our features work well across all supported devices and themes.

### Code Review
- [x] I have carefully reviewed my code and believe it is final from my side.
- [x] I have ensured there is no hardcoded color in my changes. Colors should be defined in theme files and reused.

### Theme Compatibility
- [x] I have tested the new feature or updates in both light and dark themes.
- [x] I confirm that the visual aspects of the feature are compatible and appealing in both themes.

### Device Compatibility
- [x] I have tested my changes on mobile screen sizes.
- [x] I have tested my changes on tablet screen sizes.
- [x] I have tested my changes on laptop screen sizes.

### Functional Testing
- [x] I have thoroughly tested the functionality of my changes.
- [x] I am confident that my changes do not negatively impact existing features.

### Testing Documentation
Please provide a brief description of the tests you have performed to verify your changes. Include any relevant scenarios or edge cases.

- Example test 1: Tested feature X by uploading the image
- Example test 2: Tested feature X by doing these things.

Refers: https://github.com/RobinNagpal/dodao-ui/issues/198

### CODE REVIEW

File: src/components/home/TidbitsSite/TidbitsSiteHomepage.tsx [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+export default function TidbitsSiteHomepage({ space, categoriesArray }: TidbitsSiteHomepageProps) {
+  return (
+    <div className="flex flex-col">
+      <div className="bg-white flex-1">
+        <div className="relative px-6 lg:px-8 flex flex-col justify-center">
+          <div className="inset-x-0 absolute -top-40 -z-10 transform-gpu overflow-hidden blur-3xl sm:-top-80" aria-hidden="true"></div>
+          <div className="mx-auto max-w-2xl py-8 sm:py-16 lg:pt-12">
+            <div className="text-center">
+              <h1 className="text-4xl font-bold tracking-tight text-gray-900 sm:text-6xl">{space.tidbitsHomepage?.heading}</h1>
+              <p className="mt-6 text-lg leading-8 text-gray-600">{space.tidbitsHomepage?.shortDescription}</p>
```
Comment (RobinNagpal) [inline src/components/home/TidbitsSite/TidbitsSiteHomepage.tsx:31]: We need to remove color hard coding

Comment (RobinNagpal) [reply src/components/home/TidbitsSite/TidbitsSiteHomepage.tsx:31]: May be remove `-z-10`

File: src/components/home/TidbitsSite/TidbitsSiteHomepage.tsx [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+
+interface TidbitsSiteHomepageProps {
+  space: SpaceWithIntegrationsFragment;
+  categoriesArray: CategoryWithByteCollection[];
+}
+
+export default function TidbitsSiteHomepage({ space, categoriesArray }: TidbitsSiteHomepageProps) {
+  return (
+    <div className="flex flex-col">
+      <div className="bg-white flex-1">
```
Comment (RobinNagpal) [inline src/components/home/TidbitsSite/TidbitsSiteHomepage.tsx:25]: For empty case add a fallback 'Heading' and 'Description'

File: src/components/spaces/Edit/TidbitsHomepage/UpdateTidbitsHomepageModal.tsx [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+            />
+          </div>
+        </div>
+
+        <div className="mt-6 flex items-center justify-end gap-x-6">
+          <Button
+            variant="contained"
+            primary
+            loading={updating}
+            onClick={async () => {
```
Comment (RobinNagpal) [inline src/components/spaces/Edit/TidbitsHomepage/UpdateTidbitsHomepageModal.tsx:39]: disableButton if heading and shortDescription not entered
